youtube_dl/extractor/ustream.py

```diff
@@ -72,7 +72,7 @@ class UstreamIE(InfoExtractor):
     @staticmethod
     def _extract_url(webpage):
         mobj = re.search(
-            r'<iframe[^>]+?src=(["\'])(?P<url>http://www\.ustream\.tv/embed/.+?)\1', webpage)
+            r'<iframe[^>]+?src=(["\'])(?P<url>https?://www\.ustream\.tv/embed/.+?)\1', webpage)
```

Comment:
Make `https?:` part optional.

Reply:
so something like `www.ustream.tv/embed/adsda` should work?

Reply:
No.

Reply:
then what do you mean by making the `https:?` part optional as the `?` will make it compatible for both `http` and `https`

Reply:
Re-read my post with eyes please.

Reply:
sorry but I am still unable to get what the requirement is, can you please explain with an example.

Reply:
the whole thing i guess, like e.g. in (from vimeo.py line 457):
r'<video[^>]+src=(["\'])(?P<url>(?:https?:)?//(?:www\.)?vimeo\.com/[0-9]+)\1'